### PR TITLE
Introducing currencies constants

### PR DESCRIPTION
**Pull request by wjrosa**
## Changes proposed in this Pull Request:

This is another simple refactor PR introducing a new constant class to replace all references to currency codes. This is useful to centralize information, make it easier to reference currencies, and later build specific logic for groups of currencies.

## Testing instructions

Code review should be enough. Check if the tests are still passing. No behavior should change with this PR.

---

-   [x] Covered with tests (or have a good reason not to test in description ☝️)
-   [x] Added changelog entry **in both** `changelog.txt` and `readme.txt` (or does not apply)
-   [x] Tested on mobile (or does not apply)

**Post merge**

-   [ ] Added testing instructions to the [Release Testing Instructions wiki page](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Release-Testing-Instructions) (or does not apply)
-   [ ] Added the [needs docs label](https://github.com/woocommerce/woocommerce-gateway-stripe/labels?q=docs) (or does not apply)
-   [ ] Included this PR in the Release Thread scope (or does not apply)

### CODE REVIEW

File: includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php [hunk 1]
```diff
@@ -112,7 +125,7 @@ public function get_supported_currencies() {
 
 		// Stripe account in EEA + UK and Switzerland can present the following as store currencies.
 		// EEA currencies can only transact with countries where that currency is the standard currency.
-		return [ 'CHF', 'CZK', 'DKK', 'EUR', 'GBP', 'NOK', 'PLN', 'SEK' ];
+		return [ WC_Stripe_Currency_Code::SWISS_FRANC, WC_Stripe_Currency_Code::CZECH_KORUNA, WC_Stripe_Currency_Code::DANISH_KRONE, WC_Stripe_Currency_Code::EURO, WC_Stripe_Currency_Code::POUND_STERLING, WC_Stripe_Currency_Code::NORWEGIAN_KRONE, WC_Stripe_Currency_Code::POLISH_ZLOTY, WC_Stripe_Currency_Code::SWEDISH_KRONA ];
```
Comment (annemirasol) [inline includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php]: What do you think about breaking up this now-superlong line into multiple lines?

File: includes/payment-methods/class-wc-stripe-upe-payment-method-afterpay-clearpay.php [hunk 2]
```diff
@@ -19,7 +19,7 @@ public function __construct() {
 		$this->stripe_id                    = self::STRIPE_ID;
 		$this->title                        = __( 'Clearpay / Afterpay', 'woocommerce-gateway-stripe' );
 		$this->is_reusable                  = false;
-		$this->supported_currencies         = [ 'USD', 'CAD', 'GBP', 'AUD', 'NZD' ];
+		$this->supported_currencies         = [ WC_Stripe_Currency_Code::UNITED_STATES_DOLLAR, WC_Stripe_Currency_Code::CANADIAN_DOLLAR, WC_Stripe_Currency_Code::POUND_STERLING, WC_Stripe_Currency_Code::AUSTRALIAN_DOLLAR, WC_Stripe_Currency_Code::NEW_ZEALAND_DOLLAR ];
```
Comment (annemirasol) [inline includes/payment-methods/class-wc-stripe-upe-payment-method-afterpay-clearpay.php]: What do you think about breaking up this now-superlong line into multiple lines?

**Review by annemirasol — APPROVED** [review]
- Code looks good ✨  Some super long lines may benefit from getting broken into several, for readability 🪙🪙  
- Quick smoke tests reveal no errors 👍 
- Regex search for `'[A-Z]{3}'`: no remaining raw codes for PHP files, some for JS files (expected) 👍

File: includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php (hunk 1 above)
Comment (wjrosa) [reply includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php]: Thanks for your review, Anne! I have updated both currency lists in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3459/commits/3fcebaa1f4282aed05d73be75c39c7da6b374447

**Pull request merged**